config/settings.yml

```
@@ -0,0 +1,3 @@
+validates_presence: true 
+validates_length: 2
```

đặt lại tên biến, khó hiểu em nà 
```suggestion
user:
  password:
    min_length: 2

```